localstack/config.py

```diff
@@ -32,6 +32,115 @@
 load_start_time = time.time()
 
 
+class Directories:
+    """
+    Holds the different directories available to localstack. Some directories are shared between the host and the
+    localstack container, some live only on the host and some only in the container.
+
+    Attributes:
+        infra:      container only; static infrastructure: binaries and libraries packaged with the image
```

Comment:
Actually, I think the `infra` folder is a subtype of the cache. It just happens to be pre-seeded when building the image (basically to have cache-hits on the first-run), but it will also be loaded on-demand if any of it isn't there when needed. So I think a more proper description would be "runtime only / individual; static infrastructure: binaries and libraries packaged with the image or loaded on-demand."
If my assumption here is correct it might as well just be a subfolder of the cache (since newly fetched infra content should persist across runs and reboots).

Reply:
i agree with the semantics. is just a question of what can you mount into the container. if the static infra is a subtype of cache, you cannot mount in the cache directory from the host, as that would overwrite the subdirectory existing in the container, right?